inst/Description.qml

```diff
@@ -6,7 +6,7 @@ Description
 	name		: "jaspDescriptives"
 	title		: qsTr("Descriptives")
 	description	: qsTr("Explore the data with tables and plots")
-	version			: "0.18.3"
+	version		: "0.19.0"  // Built with the JASP nightly: macOS arm64 from 06:33 21-03-2024
```

Comment:
not sure how much is this comment relevant

Reply:
yes, I removed it!